keps/sig-auth/3221-structured-authorization-configuration/README.md

```diff
@@ -435,9 +445,11 @@ the version supported by a webhook has to be mentioned using a required field
 
 The user can define a CEL expression to determine whether a request needs to dispatched
 to the authz webhook for which the expression has been defined. The user would have access
-to a `request` variable containing a `SubjectAccessReview` object in the `v1` version. If
-the version specified by `subjectAccessReviewVersion` in the request variable is `v1beta1`,
-the contents would be converted to the `v1` version before evaluating the CEL expression.
+to a `request` variable containing a `SubjectAccessReview` object in the version specified
+by the `matchConditionSubjectAccessReviewVersion` field. If the version specified by
+`subjectAccessReviewVersion` in the request variable is `v1beta1`, the contents would be
+converted to the version specified in `matchConditionSubjectAccessReviewVersion` before
```

Comment:
are we converting the subjectAccessReview object automatically to the version in matchConditionSubjectAccessReviewVersion or do we want to return a validation error if matchConditionSubjectAccessReviewVersion and subjectAccessReviewVersion don't match?

Reply:
They're independent and don't have to match. You can send v1beta1 or v1 to the webhook. You can currently only set v1 for match conditions